onnxruntime/test/framework/sparse_kernels_test.cc

```diff
@@ -150,9 +150,9 @@ This operator constructs a sparse tensor from three tensors that provide a COO
     KernelDefBuilder def;
     def.SetName(SparseFromCOO::OpName())
         .TypeConstraint("T1", DataTypeImpl::GetTensorType<int64_t>())
-        .TypeConstraint("T2", DataTypeImpl::GetTensorType<int64_t>())
+        .TypeConstraint("T2", DataTypeImpl::GetTensorType<int64_t>());
 #if !defined(DISABLE_SPARSE_TENSORS)
-        .TypeConstraint("T", DataTypeImpl::GetSparseTensorType<int64_t>());
+    def.TypeConstraint("T", DataTypeImpl::GetSparseTensorType<int64_t>());
 #endif
```

Comment:
how about moving this up so you don't have to deal with the semicolon differently

Reply:
ok, moved